src/System.Buffers/src/System/Buffers/ArrayPoolEventSource.cs

```diff
@@ -22,7 +21,7 @@ internal enum BufferAllocationReason : int
             PoolExhausted
         }
```

Comment:
While we're here should we remove the explicit name/guid from the event source? 

Perfview's docs told me this
> However it is strongly encouraged that programmers don't specify an explicit GUID but let a GUID be generated from the EventSource's name using a web-standard procedure 

But I'm hardly an expert so do whatever the right thing is!

Also, the name of the class doesn't match the name of the source so it's a little gotcha-ey.